### PR TITLE
Fix: Readme

### PR DESCRIPTION
**Pull request by CodeWithEmad**
This will introduce the following changes:

- typo fixed: transiflex -> transifex
- code-block added
- heading symbol length
- make heading dynamic for xblock cookiecutter
- Old URL with 404 response updated

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @CodeWithEmad!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads





#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.



#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?



This repository is currently unmaintained.

To get help with finding a technical reviewer, tag the community contributions project manager for this PR in a comment and let them know that your changes are ready for review:

1. On the right-hand side of the PR, find the Contributions project, click the caret in the top right corner to expand it, and check the "Primary PM" field for the name of your PM.
2. Find their GitHub handle [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3548807177/Community+Contributions+Project+Manager#Current-OSPR-Project-Managers).


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

**Comment by itsjeyd** [comment]
@CodeWithEmad Thank you for this contribution!

It looks like the tests will need a bit of work.

Once they're green we can mark this PR as ready for review.

**Comment by CodeWithEmad** [comment]
@itsjeyd Failed job has been resolved.

**Comment by itsjeyd** [comment]
Thanks @CodeWithEmad, I'll mark this PR as ready for review now.

File: cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst
Comment (OmarIthawi) [inline cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst:113]: Please link to https://docs.openedx.org/en/latest/developers/how-tos/enable-translations-new-repo.html instead.

File: cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst
Comment (OmarIthawi) [inline cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst:130]: Same here, `make push_translations` no longer exists as of OEP-58.

File: cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst
Comment (OmarIthawi) [inline cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst:140]: Same here.

File: cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst
Comment (OmarIthawi) [inline cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst:25]: This is only applicable for non-openedx XBlocks.

github.com/openedx XBlocks has full `atlas` support: 

And this link should be used instead:

 - https://docs.openedx.org/en/latest/developers/how-tos/enable-translations-new-repo.html

**Review by OmarIthawi — CHANGES_REQUESTED** [review]
Thanks @CodeWithEmad, please check the comments regarding [OEP-58](https://docs.openedx.org/en/latest/developers/concepts/oep58.html) updates

File: cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst
Comment (CodeWithEmad) [reply cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst:25]: You're right.

File: cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst
Comment (CodeWithEmad) [reply cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst:130]: Oh, so I'll remove all references to `push_translations` command.

File: cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst
Comment (CodeWithEmad) [reply cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst:140]: I don't get what you mean by "same here".

File: cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst
Comment (OmarIthawi) [reply cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst]: Please link to https://docs.openedx.org/en/latest/developers/how-tos/enable-translations-new-repo.html instead.

File: cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst
Comment (OmarIthawi) [reply cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst]: Please link to https://docs.openedx.org/en/latest/developers/how-tos/enable-translations-new-repo.html instead.

**Review by OmarIthawi — APPROVED** [review]

**Comment by itsjeyd** [comment]
Thanks for providing a review for this PR @OmarIthawi!

@CodeWithEmad Please have a look at the failing tests when you can.

**Comment by CodeWithEmad** [comment]
My changes were pretty harmless and should not cause the tests to fail, @itsjeyd. 
This is happening on other PRs like https://github.com/openedx/edx-cookiecutters/pull/488 too.

**Comment by itsjeyd** [comment]
Thanks for the details @CodeWithEmad. What's the next step here, then? Should we consider this PR blocked until someone does the work that @feanil described in https://github.com/openedx/edx-cookiecutters/pull/484#issuecomment-2338434139? I'm assuming you're not planning to do it yourself, but let me know if I got the wrong impression 🙂

**Comment by CodeWithEmad** [comment]
Sorry for not getting back to you sooner, @itsjeyd. 
I didn't have any plan for this, but if no one is working on it, I can spend some time on it.

**Comment by itsjeyd** [comment]
No worries @CodeWithEmad, and thanks for the details.

I'll mark this PR as blocked on #471, then.

**Comment by itsjeyd** [comment]
@CodeWithEmad Same [note](https://github.com/openedx/edx-cookiecutters/pull/474#issuecomment-2449718289) here, #471 has been merged so this PR should be unblocked now.

**Comment by CodeWithEmad** [comment]
It's done, @itsjeyd

**Review by feanil — APPROVED** [review]

**Pull request merged**
